sdk/attestation/Azure.Security.Attestation/src/Models/TpmAttestationRequest.cs

```diff
@@ -7,13 +7,13 @@
 namespace Azure.Security.Attestation
 {
     /// <summary>
-    /// Represents a response for a TPM attestation call. See https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol  for more information.
+    /// Represents a request for a TPM attestation call. See <seealso href="https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol"/> for more information.
     /// </summary>
     [CodeGenModel("TpmAttestationRequest")]
-    internal partial class TpmAttestationRequest
+    public partial class TpmAttestationRequest
     {
         /// <summary>
-        /// Attestation Response data. See  https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol for more details
+        /// Attestation Request data. See  <seealso href="https://docs.microsoft.com/en-us/azure/attestation/virtualization-based-security-protocol"/> for more details
         /// </summary>
         public ReadOnlyMemory<byte> Data { get => Base64Url.Decode(InternalData); set => InternalData = Base64Url.Encode(value.ToArray()); }
```

Comment:
Should we consider caching the result of the Base64 decode or do we think it unlikely that callers will query the property multiple times?

Reply:
Given the API pattern expected for this API, the Data property will be queried once and only once on each call, so decoding on every access isn't the worst possible outcome.